### PR TITLE
[KAT-2298] Add how-to-template and style notes

### PR DESCRIPTION
**Pull request by JamesFCoffey**
This PR works together with https://github.com/KatanaGraph/katana-enterprise/pull/2340

How-to template is based on the template and about file from The Good Docs Project: https://github.com/thegooddocsproject/templates

### CODE REVIEW

File: docs/contributing/how_to_template.ipynb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    "\n",
+    "This template is for a How-to guide that takes the reader through a detailed series of steps required to do an individual task/procedure. Download the guide as a Jupypter notebook, delete the note cell, and follow the section structure / [style](documentation.rst#style).\n",
+    "\n",
+    "**Title**\n",
+    "\n",
+    "Replace \"How-to template\" with the name you want to publish.\n",
+    "\n",
+    "**Before you start**\n",
+    "\n",
+    "This section prevents readers from getting halfway through and discovering that they need to go and read other documentation before they can continue. Prerequisites can include other articles or information to read, or it can be technical dependencies such as requiring an internet connection, or particular software.\n",
```
Comment (origandrew) [inline docs/contributing/how_to_template.ipynb:28]: Put this content in the actual section

File: docs/contributing/how_to_template.ipynb [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3da983f5",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
```
Comment (origandrew) [inline docs/contributing/how_to_template.ipynb:18]: I don't think there is a need for a note. Make "Step One" in the how-to template something like

## Setting up the how-to guide
1. filename and location ...
2. Replace the title ...
3. Fill out the "Before you start" section ...

File: docs/contributing/how_to_template.ipynb [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    "\n",
+    "This section prevents readers from getting halfway through and discovering that they need to go and read other documentation before they can continue. Prerequisites can include other articles or information to read, or it can be technical dependencies such as requiring an internet connection, or particular software.\n",
+    "\n",
+    "Describe what the audience needs to know, or needs to have, before they attempt the How-to. By stating the requirements up-front, you prevent your readers from having a bad experience with your How-to. You must include links to procedures or information about how to get what they need. If not possible, give useful pointers.\n",
+    "\n",
+    "Delete this section if your readers can go to the steps without requiring any prerequisite knowledge/tools.\n",
+    "\n",
+    "**Step-by-step guide**\n",
+    "\n",
+    "When you are explaining steps in a process, it can be useful to include screenshots for each key part of the process. This can help readers orientate themselves as they move through the steps. It can also help someone who is evaluating the software see how it works without having to install it.\n",
```
Comment (origandrew) [inline docs/contributing/how_to_template.ipynb:36]: Make this content in step two:
1. explain steps and process

File: docs/contributing/how_to_template.ipynb [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+    "\n",
+    "Brief instructions explaining how to interpret the image. When an image, such as a screenshot, is quicker to interpret than descriptive text, put the screenshot first, otherwise lead with the text."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "916b7653",
+   "metadata": {},
+   "source": [
+    "### Title for step (ordered list variant)"
```
Comment (origandrew) [inline docs/contributing/how_to_template.ipynb:104]: Add boilerplate setup section

**Review by origandrew — COMMENTED** [review]
Some comments

**Comment by JamesFCoffey** [comment]
Replaced by https://github.com/KatanaGraph/katana/pull/867

**Pull request closed without merging**